### PR TITLE
Added Register Feature

### PR DESCRIPTION
**Pull request by ivanlink92**
1. Code Quality

- [ ] Is the code easy to read and understand?

- [ ] Are functions/methods short, clear, and focused on a single task?

- [ ] Is the code modular and reusable where appropriate?

2. Functionality

- [ ] Does the code meet all the functional requirements?

- [ ] Are all edge cases and error conditions handled?

- [ ] Does it integrate correctly with existing code?

3. Style & Conventions

- [ ] Does the code follow the team’s style guide (e.g., PEP8 for Python, Airbnb for JavaScript)?

- [ ] Are naming conventions consistent and meaningful?

- [ ] Are indentation and spacing consistent?

4. Testing

- [ ] Are there unit and/or integration tests for new features?

- [ ] Do the tests cover both normal and edge cases?

- [ ] Do all tests pass?

5. Security

- [ ] Are all user inputs properly validated and sanitized?

- [ ] Are there any hardcoded credentials or secrets?

- [ ] Is sensitive data handled securely?

6. Performance

- [ ] Is the code efficient and optimized where needed?

- [ ] Are there any unnecessary computations or database queries?

- [ ] Are large datasets handled properly?

7. Documentation

- [ ] Are code comments used where necessary to explain complex logic?

- [ ] Are public methods and APIs documented?

- [ ] Is the README or other project documentation updated (if needed)?

### CODE REVIEW

File: src/components/Register.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      />
+      <div className="main">
+        <input type="checkbox" id="chk" aria-hidden="true" />
+        <div className="signup">
+          <form onSubmit={handleRegister}>
+            <label htmlFor="chk" aria-hidden="true">
+              Register
+            </label>
+            <input type="text" name="username" placeholder="Username" required />
+            <input type="text" name="password" placeholder="Password" required />
```
Comment (goonle) [inline src/components/Register.js:39]: use type password instead of text
```suggestion
            <input type="password" name="password" placeholder="Password" required />
```

Comment (ivanlink92) [reply src/components/Register.js]: It fixed!

**Review by goonle — APPROVED** [review]
I checked it. Well done.
